src/test/java/org/opentripplanner/transit/speed_test/SpeedTest.java

```diff
@@ -61,6 +63,11 @@ public class SpeedTest {
   private SpeedTest(SpeedTestCmdLineOpts opts) {
     this.opts = opts;
     this.config = SpeedTestConfig.config(opts.rootDir());
+
+    var features = new OtpConfigLoader(opts.rootDir()).loadOtpConfig();
```

Comment:
@slvlirnoff I think you were asking for this feature, weren't you?

Reply:
Yes! this is nice